registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -662,9 +658,16 @@ private void validateBiometricRecord(BiometricRecord biometricRecord, List<Strin
 		boolean isBioFound = false;
 		for (String biometricSegment : biometricModalitySegmentsMap.keySet()) {
 			if (!modalities.contains(biometricSegment)) {
-				throw new DataShareException("Biometrics Segments Not Configured for modality : " + biometricSegment);
+				throw new DataShareException(
+						"Modalities  not Configured as per policy for modality : " + biometricSegment);
 			}
+			List<String> biometricModalitySegmentsMapFromPolicy = typeAndSubtypMap.get(biometricSegment);

```

Comment:
rename biometricModalitySegmentsMapFromPolicy to policySegmentList

Reply:
fixed